resources/views/scheduleInstrument.html

```diff
@@ -0,0 +1,427 @@
+<div id="topContainer">
+    <div id="projectDropDown"></div>
+    <br/>
+    <div id="instrumentDropDown"></div>
+    <br/>
+    <div id="instrumentOperatorDropDown"></div>
+    <br/>
+    <div id="paymentMethodDropDown"></div>
+    <a href="javascript:void(0);" id="addPaymentMethod">[Add Payment Method]</a>
```

Comment:
I think browsers treat this as inline script and it'll be rejected by our stronger CSP. Do you see any browser console errors? You may need to register this dynamically.

Reply:
changed